src/App.css

```
@@ -7,11 +7,10 @@
   font-family: 'Arial' sans-serif;
 }
 
-.budgets, .expenses, .home, .loans-debts, .savings, .shared-expenses, .wallets {
+.budgets, .expenses, .home, .loans-debts, .savings, .shared-expenses {
```

Esto lo podriamos dejar asi, para que sea mas entendible

```suggestion
.budgets, 
.expenses, 
.home, 
.loans-debts, 
.savings, 
.shared-expenses {
```